src/models/contact.js

```diff
@@ -364,6 +364,31 @@ export default class Contact {
 		this.vCard.updatePropertyWithValue('org', org)
 	}
 
+	/**
+	 * Return the first orgManager
+	 *
+	 * @readonly
+	 * @memberof Contact
+	 */
+	get orgManager() {
+		return this.firstIfArray(this.vCard.getFirstPropertyValue('orgmanager'))
```

Comment:
is this possibly a standardized property? I'm not able to find references of ORGMANAGER in the vcard specs. If it's not, should be prefix with `X-` @skjnldsv?

Reply:
Yep, if not official, prefix is mandatory

Reply:
If you use X-ManagersName, it will be compatible with /kalendar, better than nothing 😜

A similar concept exist in google eith manager and assistant name but they don't even use vcards

Reply:
We already have "Relationship to you" with "Manager", "Colleague" and "Assistant", is that not a standard of sorts we can use here too?
![image](https://user-images.githubusercontent.com/925062/185460651-f1641592-8453-4ce0-be02-765c30fe7637.png)

Reply:
I just pushed commit with all the requested changes/bugs/etc.
Currently I'm using X-ManagersName.. Relationship field is like first person property and I'm not sure(never used) if is anything else except an info.. except if extend/change it..

Reply:
*Relationship to you* is not helpful because it only stores the type of relation and is always relative to the addressbook owner. E.g. in your addressbook there might be a contact for me and it says *Colleague*. Nothing more, nothing less.

`X-ManagersName` seems a lot more appropriate. The only issue we just found out is that KOrganizer stores the *name* of the other person for this field, not the UID. So it is not a reference to another contact. It is just a value. And this value is prone for conflicts in larger organizations where names are not unique.

I'll see if we can put together some kind of hybrid solution where we store the UID for our purposes but also X-ManagersName for interoperability.

Reply:
We will go with something like `X-MANAGERSNAME;UID=abc-def-123:John`